docs/user/introduction/information.rst

```diff
@@ -104,22 +93,17 @@ Facebook
 - **Poland:** https://www.facebook.com/Dash.Polska
 - **Russia:** https://www.facebook.com/Dash.Russia
 - **Thailand:** https://www.facebook.com/groups/dashembassythailand
-- **Venezuela:** https://www.facebook.com/groups/DarkcoinVenezuela
 - **Vietnam:** https://www.facebook.com/dashvietnam
 
 
 Twitter
 -------
 
 - **Dash Official Account:** https://twitter.com/dashpay
-- **Ryan Taylor, CEO of Dash Core Group:** https://twitter.com/RTaylor05
```

Comment:
Should we put Sam's here? https://twitter.com/SamWestrich/

Reply:
added and pushed